features/provider/stats/application.feature

```diff
@@ -4,16 +4,41 @@ Feature: Application stats
   I want to see this data in a chart
 
   Background:
-    Given a provider has a developer "Slartibartfast" with an application name "Aircar"
+    Given a provider is logged in
+    And the provider has a buyer with an application
     And all the rolling updates features are off
 
     @javascript
     Scenario: Developer's Application chart has data
-      Given buyer "Slartibartfast" made 2 service transactions 12 hours ago:
+      Given the buyer made 2 service transactions 12 hours ago:
         | Metric   | Value |
         | hits     |    20 |
-
-      When the provider is logged in and visits the "Aircar" application stats
+      When they go to the provider application page
+      And they follow "Analytics"
       Then there should be a c3 chart with the following data:
         | name          | total  |
         | Hits          | 40   |
+
+  @javascript
+  Scenario: Developer's Current Utilization section has data
+    Given the provider has a free application plan "Limited"
+    And an usage limit on plan "Limited" for metric "hits" with period hour and value 100
+    And the buyer changed to application plan "Limited"
+    And the buyer makes 2 service transactions with:
+      | Metric   | Value |
+      | hits     |    20 |
+    And the backend responds to a utilization request for the application with:
```

Comment:
IMO, we should ideally abstract the exact response from Backend, and hide it within the steps (cucumbers are more "business rules", so I guess we shouldn't get too technical). So maybe in the scenario make it a table:
```
      | period   | metric_name | max_value | current_value
      | minute   |    hits     | 100       | 40
```
and in the step transform it to JSON.

But I think it's probably fine :)

Reply:
Done!